HaikuPorter/Port.py

```diff
@@ -1212,9 +1212,23 @@ def _updateShellVariables(self, forParsing):
 				for k, v in configureDirs.items())
 
 		# add one more variable containing all the dir args for CMake:
-		cmakeDirArgs = {}
-		for k, v in configureDirs.items():
```

Comment:
looks like a simpler fix would be replacing configureDirs with relativeConfigureDirs? It would avoid duplicating all the paths here.

Reply:
it's not exactly relativeConfigureDirs because we also have prefix and sysconfdir which are absolute